python/tests/integration/arcticdb/test_real_storage.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import os
+import pandas as pd
+from arcticdb.arctic import Arctic
+
+REAL_STORAGE_TESTS_ENABLED = True if os.getenv("ARCTICDB_REAL_STORAGE_TESTS") == "1" else False
+REAL_STORAGE_LIB_NAME = os.getenv("ARCTICDB_REAL_STORAGE_LIB_NAME")
+
+
+if REAL_STORAGE_TESTS_ENABLED:
+    # TODO: Maybe add a way to parametrize this
+    LIBRARIES = [
+        # LINUX
+        "linux_3_6",
+        "linux_3_7",
+        "linux_3_8",
+        "linux_3_9",
+        "linux_3_10",
+        "linux_3_11",
+        # WINDOWS
+        "windows_3_7",
+        "windows_3_8",
+        "windows_3_9",
+        "windows_3_10",
+        "windows_3_11",
+    ]
+else:
+    LIBRARIES = []
+
+
+# TODO: Add a check if the real storage tests are enabled
+@pytest.mark.parametrize("library", LIBRARIES)
+def test_real_s3_storage_read(real_s3_credentials, library):
+    endpoint, bucket, region, access_key, secret_key, clear = real_s3_credentials
+    uri = f"s3s://{endpoint}:{bucket}?access={access_key}&secret={secret_key}&region={region}&path_prefix=ci_tests/"
+    ac = Arctic(uri)
+    lib = ac[library]
+    symbols = lib.list_symbols()
+    assert len(symbols) == 3
+    for sym in ["one", "two", "three"]:
+        assert sym in symbols
+    for sym in symbols:
+        df = lib.read(sym).data
+        column_names = df.columns.values.tolist()
+        assert column_names == ["x", "y", "z"]
+
+
+def test_real_s3_storage_write(real_s3_credentials, three_col_df):
+    library_to_write_to = REAL_STORAGE_LIB_NAME
```

Comment:
Isn't this the same library name as the one used by the seed jobs? Don't we want to write to a completely new library here (and use your new fixture to clean it up)?